ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -273,27 +286,25 @@ def update_subscription(self, channel):
         """
         subscription = OCP(
             resource_name=constants.OCS_SUBSCRIPTION,
-            kind='subscription',
-            namespace=config.ENV_DATA['cluster_namespace'],
+            kind="subscription",
+            namespace=config.ENV_DATA["cluster_namespace"],
         )
-        current_ocs_source = subscription.data['spec']['source']
-        log.info(
-            f"Current OCS subscription source: {current_ocs_source}"
-        )
-        ocs_source = current_ocs_source if self.upgrade_in_current_source else (
-            constants.OPERATOR_CATALOG_SOURCE_NAME
+        current_ocs_source = subscription.data["spec"]["source"]
+        log.info(f"Current OCS subscription source: {current_ocs_source}")
+        ocs_source = (
+            current_ocs_source
```

Comment:
not sure why split that line

Reply:
black is splitting up expressions like this into logical parts since a one-liner is too long.